msvc/vsimporter-templates/WinStore10-App/Package-native.appxmanifest

```diff
@@ -8,7 +8,7 @@
 
   <Identity
     Name="$guid9$"
-    Publisher="islandwood"
+    Publisher="CN=islandwood"
```

Comment:
CN= format has always worked in the past I can see several examples of this in our internal codebase, so this should pose any compat issues.

---
In reply to: [83706840](https://github.com/Microsoft/WinObjC/pull/1163#discussion_r83706840) [](ancestors = 83706840)